server/block/slime.go

```diff
@@ -0,0 +1,21 @@
+package block
+
+// Slime is a translucent block that entities can bounce on.
+type Slime struct {
```

Comment:
This also also should implement `block.EntityLander`, and should reset the entity's fall distance if it implements `FallDistanceEntity`